### DISCUSSION TITLE
GitHub Desktop indicating that there were 1300+ new changed files but now displays only two changes

### DISCUSSION THREAD

**Discussion by fastbrother** [community/community#102687] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Feature Area

Issues

### Body

Hello, as the title says, I was having issues with GitHub detecting the creation of 1300+ files when I only made a couple small changes in Unity. I am using a .gitignore, which has been working fine for me and I've already made 80+ successful commits in the past to this specific repository. 

Upon closer inspection, I noticed no difference between the files Github was identifying as new and the files already present in my repository. In attempting to take a screenshot of this abnormal behavior, I pressed command shift s while GitHub was open (forgot the shortcut for taking a screenshot) and accidentally stashed all the 1300+ files. After pressing restore, GitHub now only identifies two files as having been changed. 

Could anyone explain what happened? I'm confused as to whether I should consider this issue resolved and continue pushing/committing as normal. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by darrehanrasool** [comment]
You have initialized repository in subfolders i think.

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @fastbrother ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Projects and Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Comment by MrLoadex** [comment]
Hello!

**Comment by ShutovKS** [comment]
Most likely the problem is that you changed the name of some parent folder, which led to such huge changes

**Comment by tigran-sargsyan-w** [comment]
Perhaps the large number of created files in GitHub when making small changes in the Unity project is due to the peculiarities of Unity itself. The platform may create or modify numerous files within the project even with minor alterations.

I believe the issue with .gitignore might stem from some files that should have been ignored still making their way into the commit. This could occur due to changes in dependencies or Unity settings not accounted for in the .gitignore file.

It's peculiar that after accidentally stashing and then restoring all 1300+ files, GitHub now only recognizes two changed files. It's possible that restoring the stash reverted the repository to the state before stashing, disregarding any minor changes that had already been committed.

It appears that the issue is resolved as GitHub now only displays two changed files. However, it's advisable to carefully review the project's status and commits to ensure all changes are properly saved. Additionally, consider updating the .gitignore file to ensure all unnecessary files and directories are correctly ignored in commits.
